src/App.tsx

```diff
@@ -1,26 +1,19 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import './App.scss';
 
-function App() {
+import Header from './components/Header/Header';
+
+import Courses from './components/Courses/Courses';
+
+import { mockedCoursesList } from './constants';
+
+export default function App() {
 	return (
-		<div className='App'>
-			<header className='App-header'>
-				<img src={logo} className='App-logo' alt='logo' />
-				<p>
-					Edit <code>src/App.tsx</code> and save to reload.
-				</p>
-				<a
-					className='App-link'
-					href='https://reactjs.org'
-					target='_blank'
-					rel='noopener noreferrer'
-				>
-					Learn React
-				</a>
-			</header>
-		</div>
+		<>
+			<Header />
+			<div className='content-block'>
+				<Courses data={mockedCoursesList} />
```

Comment:
As we discussed today, you have two options:
1) Move state with courses to the App component from the Courses component and pass the value and the setter to Courses component as props

```suggestion
const [courses, setCourses] = useState(mockedCoursesList);
....
				<Courses courses={courses} updateCourses={setCourses}/>
```
2) Remove `data` prop,  import `mockedCoursesList` in the Courses and put it in the state there. So the same line
```
const [courses, setCourses] = useState(mockedCoursesList);
```
but in the Courses component